packages/tasit-action/src/erc721/ERC721Full.test.js

```diff
@@ -188,9 +196,7 @@ describe("TasitAction.ERC721.ERC721Full", () => {
 
       const contractErrorListener = message => {
```

Comment:
Maybe we should call this param something other than `message` because the `error` within `message` contains `message`, which is confusing

Reply:
Should we change the API from error listeners?


```javascript
// Subscription._emitErrorEvent(error):
// from
    const message = { error };
    errorEventListener.listener(message);
// to
    errorEventListener.listener(error);
```

Reply:
Hmm, interesting. Yeah, I'd be open to that if you think that's better. Can you think of any other information we might want to decorate the object with that wouldn't fit in the error object?

What if we wanted to give our own name for the error but still pass through the original error name? Is our current approach better for that?

Reply:
I think that we can extend the original error if we want to add more info to that sometime.
I've written a sample: https://es6console.com/jv3m7125/

Reply:
Yeah, that's perfect

Reply:
Feel free to resolve this and create an issue for that?

The idea of renaming the param, which is the original subject of this thread, can be resolved

Reply:
> Feel free to resolve this and create an issue for that?
It was partially solved w/ the PR https://github.com/tasitlabs/TasitSDK/pull/369.
Missing using inherit to decorate the error object. Extracted to: https://github.com/tasitlabs/TasitSDK/issues/370
Resolving this.

Reply:
Perfect, thanks!